apps/test/unit/lib/levelbuilder/lesson-editor/ActivitiesEditorTest.jsx

```diff
@@ -6,31 +6,21 @@ import {UnconnectedActivitiesEditor as ActivitiesEditor} from '@cdo/apps/lib/lev
 import {sampleActivities} from './activitiesTestData';
 
 describe('ActivitiesEditor', () => {
-  let defaultProps, addActivity;
+  let defaultProps, addActivity, serializeActivities;
   beforeEach(() => {
     addActivity = sinon.spy();
+    serializeActivities = sinon.spy();
     defaultProps = {
       activities: sampleActivities,
-      addActivity
+      addActivity,
+      serializeActivities
     };
   });
 
   it('renders default props', () => {
     const wrapper = shallow(<ActivitiesEditor {...defaultProps} />);
     expect(wrapper.find('button').length).to.equal(1);
     expect(wrapper.find('ActivityCardAndPreview').length).to.equal(1);
-
-    const hiddenInputs = wrapper.find('input[type="hidden"]');
```

Comment:
@davidsbailey because of moving the serializeActivities method out of this component this check no longer makes sense here. Should I be trying to check this in LessonEditor?

Reply:
Hey Dani, it seems fine to move this check up to the lesson editor test. however, I am a bit confused because I still see the input in ActivitiesEditor.jsx:
![Screen Shot 2020-11-10 at 8 03 21 PM](https://user-images.githubusercontent.com/8001765/98764065-de314b00-238f-11eb-9bc6-6348f89480b8.png)

Reply:
If you're saying that you want to move this hidden input up to LessonEditor, then yes, I think it makes sense to do that, and move this corresponding check into LessonEditorTest.

Reply:
I think I solved my own problem.

Reply:
once we get rid of the form post, we should be able to remove the hidden input. it should also be possible to remove the `name` attribute from any input fields which were being used to name the data as part of the form post.